src/components/share/index.tsx

```diff
@@ -67,30 +66,7 @@ export const ShareControl: FC = () => {
               <PiLinkSimpleBold className={CONTROL_ICON_STYLES.default} />
               <span className="text-xs">Copy URL link</span>
             </button>
-            <div className="flex items-center space-x-2">
-              <TwitterShareButton
-                url={urlCopy}
-                // TODO: update title
-                title="Open Earth Monitor Cyberinfrastructure"
-                aria-label="share twitter"
-              >
-                <div className={cn(CONTROL_BUTTON_STYLES.default, 'h-[28px] w-[28px]')}>
-                  <RiTwitterXLine className={CONTROL_ICON_STYLES.default} />
-                </div>
-              </TwitterShareButton>
-
-              <LinkedinShareButton
-                url={urlCopy}
-                // TODO: update title
-                title="Open Earth Monitor Cyberinfrastructure"
-                className="align-baseline"
-                aria-label="share in linkedin"
-              >
-                <div className={cn(CONTROL_BUTTON_STYLES.default, 'h-[28px] w-[28px]')}>
-                  <RiLinkedinFill className={CONTROL_ICON_STYLES.default} />
-                </div>
-              </LinkedinShareButton>
-            </div>
+            <SocialMedia url={urlCopy} />
```

Comment:
I understand this decision by reusability, but I think in this case you are asumming we will have the same social networks in the share and in the footer. Also, they have different functionality. One will share the url, and the another one it's just a link. I think in this case only ensure the icons are the same but not re-use them.